libcloudforensics/providers/gcp/internal/monitoring.py

```diff
@@ -91,6 +91,90 @@ def ActiveServices(self, timeframe: int = 30) -> Dict[str, int]:
               ret[service] = int(val)
     return ret
 
+
+  def GetNetworkData(
+      self, instance_ids: Optional[List[str]] = None,
+      days: int = 2) -> List[Dict[str, Any]]:
+    """Returns outbound network metrics for compute instances.
+
+    By default returns usage for the last two days for the given instance.
+
+    Args:
+      instance_ids list[str]: Optional. A list of instance IDs to collect
+        metrics for. When not provided will collect metrics for all instances
+        in the project.
+      days (int): Optional. The number of days to collect metrics for.
+
+    Returns:
+      List[Dict[str, Any]]: a list of network usage for each instance
+      in the format
+        [
+          {
+            'instance_name': str,
+            'network_usage':
+            [
+              {
+                'timestamp': str,
+                'bytes': float
+              },
+            ]
+          },
+        ]
+    """
+    service = self.GcmApi()
+    gcm_timeseries_client = service.projects().timeSeries() # pylint: disable=no-member
+    start_time = common.FormatRFC3339(
+        datetime.datetime.utcnow() - datetime.timedelta(days=days))
+    end_time = common.FormatRFC3339(datetime.datetime.utcnow())
+
+    network_filter = [
+        'metric.type=',
+        '"compute.googleapis.com/instance/network/sent_bytes_count" ',
+        'resource.type="gce_instance"'
+    ]
+
+    if instance_ids:
+      network_filter.append(
+          ' AND (resource.label.instance_id = "{0:s}"'.format(instance_ids[0]))
+      if len(instance_ids) > 1:
+        for instance_name in instance_ids[1:]:
+          network_filter.append(
+              ' OR resource.label.instance_id = "{0:s}"'.format(instance_name))
```

Comment:
Based on [0] it looks like you could simplify this as:
```suggestion
      ids = ' OR '.join(f'"id"' for id in instance_ids)
      network_filter.append(f' AND (resource.label.instance_id = ({ids}))')
```

[0] https://cloud.google.com/logging/docs/view/logging-query-language

Reply:
Probably correct but will leave it to match the existing code patterns for now